examples/kirchhoffplate.cc

```diff
@@ -51,28 +51,24 @@ int main(int argc, char **argv) {
   double lambdaLoad = 1 * std::pow(thk, 3);
 
   const int subsample = postProcessParameters.get<int>("subsample");
-  std::cout << "ALUGRID_VERBOSITY_LEVEL1: " << getenv("ALUGRID_VERBOSITY_LEVEL") << std::endl;
   // Log the Parameters
   spdlog::info(
-      "Filename: {} \n The following parameters were used: \nMaterial: E {}, nu {} \nRefinements: global {}, u {}, v "
+      "Filename: {} \nThe following parameters were used: \nMaterial: E {}, nu {} \nRefinements: global {}, u {}, v "
```

Comment:
Maybe we can throw out the dependency on spdlog replace this with std::format

Reply:
Doesn't ikarus use spdlog in the  `init.hh`?

Reply:
Ah yes, this should then be changed there also. But can you still remove it here and create an Issue at Ikarus?

Reply:
Alright!